client/src/pages/signin/Signin.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react"
+import './Signin.scss'
+import { Link } from 'react-router-dom'
+
+import TextField from '@material-ui/core/TextField';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+class signin extends Component{
+
+    constructor(){
+        super()
+
+        this.state={
+            login: {
+                email: '',
+                password: ''
+            },
+            passLength: 0,
+            snackbaropen: false,
```

Comment:
you might want to make these state keys into camelcase like `snackbarOpen` and `snackbarMsg`, it makes it easier to read :)

Reply:
thank you for pointing it out, the changes have been made